apps/web/app/s/[surveyId]/LinkSurvey.tsx

```diff
@@ -72,13 +72,13 @@ export default function LinkSurvey({ survey }: LinkSurveyProps) {
       case "notEquals":
         return answerValue !== logic.value;
       case "lessThan":
-        return answerValue < logic.value;
+        return logic.value !== undefined && answerValue < logic.value;
```

Comment:
Do this changes also need to be made in `packages/js/src/components/SurveyView.tsx`?

Reply:
@moritzrengert I think you made this change, no? Please clarify 🙌

Reply:
@moritzrengert I think you made this change. I have updated the SurveyView and tested the lessThan condition but please confirm that its correct. And why is the check for undefined needed for lessThan? And why not for the others? 🤓

Reply:
got it, thanks :)